### DISCUSSION TITLE
Export Theme to reimport to other Machine?

### DISCUSSION THREAD

**Discussion by D1Ck3n** [realmazharhussain/gdm-settings#77] [General]
Good day,

first of all: I think your tool is great! This is exactly what I was looking for! 

My question: would it be possible to export the theme what I create with the tool and then import it again on another PC?

I ask because I wrote a script that automatically sets up a Linux workstation for a user and I would like a custom theme for it.

Previously I did this with the greeter file, but since Ubuntu 20.04 you can no longer edit the login screen.

Alternatively: could your tool also be operated purely via command line? Then I could integrate it into my script without the user seeing the GUI.

Happy to hear from you!

**Comment by realmazharhussain** [comment]
> first of all: I think your tool is great! This is exactly what I was looking for!

Thank you.

> My question: would it be possible to export the theme what I create with the tool and then import it again on another PC?

Yes, just transfer the extracted theme over to the other pc.

> I ask because I wrote a script that automatically sets up a Linux workstation for a user and I would like a custom theme for it.

This app does not have that much shell theme customization options. If you actually want a custom theme that is drastically different from the default theme, you might want to search for a guide on "how to create a gnome shell theme?".

> Alternatively: could your tool also be operated purely via command line?

No. I mean it could if I added a command-line interface. But then users would expect to run it with sudo and it would cause weird inconsistencies.

I have considered adding a command-line interface in the past. Actually, it had a command-line interface in the beginning. But it causes more problems than it solves. So, I removed it.

**Reply by D1Ck3n** [reply]
Thanks a lot for the reply! :)

**Reply by D1Ck3n** [reply]
Just got back to this: where do i put the theme, when i transfered it to a other PC? also to /usr/share/themes?

**Reply by realmazharhussain** [reply]
> where do i put the theme, when i transfered it to a other PC? also to /usr/share/themes?

Yes. You can also put it in `/usr/local/share/themes` or `~/.local/share/themes`. I personally prefer `/usr/local/share/themes` over the other directories but that's just me.
